### PR TITLE
Feat/home screen

### PR DESCRIPTION
**Pull request by Leeosteves**
# 🚨 Fill in the fields below

[Create Home Screen](https://trello.com/c/pSeXrMLs/18-create-home-screen)

## Description

```
~> Added screen home
```

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

- [x] PRODUCT

  - [x] Put PR link on ticket
  - [x] Move the ticket

- [x] DEVELOPERS

  - [x] My code follows the style guidelines of this project
  - [x] I have performed a self-review of my own code
  - [x] I have made corresponding changes to the documentation
  - [x] My changes generate no new warnings and I added `TODO:`, `FIXME:`

🚨 - `Approve: Wait at least one influential in the project or more experienced`

# FRONTEND

## Screenshot or video of your application

🚨 - `Fill with your application`

https://user-images.githubusercontent.com/106574167/179492876-d3a4047f-d920-461e-8d4b-027ee034c264.mp4

### CODE REVIEW

File: src/navigation/routes/index.tsx [hunk 1]
```diff
@@ -8,10 +8,16 @@ const {Navigator, Screen} = createNativeStackNavigator();
...
-      <Screen name="Home" component={Home} />
-      <Screen name="Login" component={Login} />
+    <Navigator screenOptions={{}}>
+      <Screen
+        options={{
+          headerShown: false,
+        }}
+        name="Home"
+        component={Home}
+      />
```
Comment (IbraDajani) [inline src/navigation/routes/index.tsx:18]: remove the header from all screens in the route

File: src/screens/Home/index.tsx [hunk 2]
```diff
@@ -1,21 +1,33 @@
...
-        ItemSeparatorComponent={() => <Separator height={10} />}
-        showsVerticalScrollIndicator={false}
+      <StatusBar
+        translucent
+        backgroundColor="transparent"
+        barStyle="dark-content"
       />
+      <BackgroundImg source={pokeBallImage} />
+      <HeaderStyle />
+      <TextName>Pokedex</TextName>
```
Comment (IbraDajani) [inline src/screens/Home/index.tsx:20]: Change the name of this component to something more specif... like `HeaderTitle` or `HeaderLabel`

File: src/screens/Home/localComponents/HeaderStyle/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import Icon from '~/components/Icon';
+import {Container, RowBetweenHome} from './styles';
+
+const HeaderStyle = () => {
+  return (
+    <Container>
+      <RowBetweenHome>
+        {/* <TouchableOpacity>
+          <Icon icon="logout" size={30} activeColor="white" />
+        </TouchableOpacity> */}
```
Comment (IbraDajani) [inline src/screens/Home/localComponents/HeaderStyle/index.tsx:12]: Remove comments

File: src/screens/Home/localComponents/HeaderStyle/styles.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import Text from '../../../../components/Text';
+
+export const Container = styled.View`
+  height: 80px;
+  justify-content: center;
+  align-items: center;
+  background-color: transparent;
+  flex-direction: row;
+`;
+export const RowBetweenHome = styled(RowBetween).attrs({})`
```
Comment (IbraDajani) [inline src/screens/Home/localComponents/HeaderStyle/styles.ts:13]: Remove the attrs({ }) if you are not passing any attrs to it

File: src/screens/Home/styles.ts [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
+export const BackgroundImg = styled(Image).attrs({
+  resizeMode: 'contain',
+})`
+  width: 450px;
+  height: 450px;
+  position: absolute;
+  left: 140px;
+  bottom: 550px;
+  opacity: 0.4;
+  /* tint-color: white; */
```
Comment (IbraDajani) [inline src/screens/Home/styles.ts:32]: Remove comment

File: src/screens/Internal/index.tsx [hunk 6]
```diff
@@ -2,15 +2,16 @@ import React from 'react';
...
+import {Image, Pressable} from 'react-native';
+import {RoutePokemonDTO} from '../../@types/dtos/pokemonDTO';
 
 const Internal: React.FC = () => {
   const {item}: RoutePokemonDTO = useRoute().params;
   const {goBack} = useNavigation();
   return (
     <Container>
       <Pressable onPress={goBack}>
+        <Image style={{height: 300, width: 300}} source={{uri: item?.img}} />
```
Comment (IbraDajani) [inline src/screens/Internal/index.tsx:14]: Remove inline styling

File: src/screens/Internal/styles.ts [hunk 7]
```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components/native';
 
 export const Container = styled.Pressable`
   flex: 1;
-  justify-content: center;
+  /* justify-content: center; */
```
Comment (IbraDajani) [inline src/screens/Internal/styles.ts:5]: Remove comment

**Review by IbraDajani — CHANGES_REQUESTED** [review]

File: src/screens/Home/localComponents/HeaderLabel/styles.ts [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+import RowBetween from '~/components/RowBetween';
+
+export const Container = styled.View`
+  height: 80px;
+  justify-content: center;
+  align-items: center;
+  background-color: transparent;
+  flex-direction: row;
+`;
+export const RowBetweenHome = styled(RowBetween)`
```
Comment (IbraDajani) [inline src/screens/Home/localComponents/HeaderLabel/styles.ts:11]: Do not use RowBetween component like this

Create a View component with the same CSS + the CSS you want to add

File: src/screens/Home/styles.ts [hunk 9]
```diff
@@ -1,7 +1,32 @@
...
+  width: 160px;
+  padding-left: 20px;
+  position: absolute;
+  top: 140px;
+`;
+export const FlatListContainer = styled.View`
+  margin-top: 130px;
+`;
+
+export const BackgroundImg = styled(Image).attrs({
```
Comment (IbraDajani) [inline src/screens/Home/styles.ts:23]: if the Image is from the React native, then use it like `styled.Image` and not `styled(Image)`

**Review by IbraDajani — CHANGES_REQUESTED** [review]

File: src/navigation/routes/index.tsx (hunk 1 above)
Comment (IbraDajani) [reply src/navigation/routes/index.tsx]: Removed in commit b27b14336b1f9ef3b239369d6734b204f069d836

Comment (Leeosteves) [reply src/navigation/routes/index.tsx]: resolved in commit b27b14336b1f9ef3b239369d6734b204f069d836

**Review by IbraDajani — APPROVED** [review]

File: src/screens/Home/index.tsx (hunk 2 above)
Comment (Leeosteves) [reply src/screens/Home/index.tsx]: resolved in commit b27b14336b1f9ef3b239369d6734b204f069d836

File: src/screens/Home/localComponents/HeaderStyle/styles.ts (hunk 4 above)
Comment (Leeosteves) [reply src/screens/Home/localComponents/HeaderStyle/styles.ts]: removed in commit b27b14336b1f9ef3b239369d6734b204f069d836

File: src/screens/Home/styles.ts (hunk 5 above)
Comment (Leeosteves) [reply src/screens/Home/styles.ts]: resolved in commit b27b14336b1f9ef3b239369d6734b204f069d836

File: src/screens/Internal/styles.ts (hunk 7 above)
Comment (Leeosteves) [reply src/screens/Internal/styles.ts]: resolved in commit b27b14336b1f9ef3b239369d6734b204f069d836

File: src/screens/Home/styles.ts (hunk 9 above)
Comment (Leeosteves) [reply src/screens/Home/styles.ts]: resolved in commit 3a01a5dcc6f5c7a46c39041ba3171ccbc32361e2

File: src/screens/Home/localComponents/HeaderLabel/styles.ts (hunk 8 above)
Comment (Leeosteves) [reply src/screens/Home/localComponents/HeaderLabel/styles.ts]: resolved in commit 3a01a5dcc6f5c7a46c39041ba3171ccbc32361e2

**Pull request merged**